client/components/share/helpers.js

```diff
@@ -20,3 +20,20 @@ export const hardTruncation = limit => title => title.slice( 0, limit ).concat(
 
 export const firstValid = ( ...predicates ) => a =>
 	find( predicates, ( p => false !== p( a ) ) )( a );
+
+export const truncateArticleContent = ( maxCharacters, content ) => {
```

Comment:
I know this is very close to `truncateAtSpace` above but the that function has the odd effect of making you _guess_ the length of the last word with the `lower` param. if `lower` and `upper` are too close, you don't get any truncation. If they are too far apart you get an unnecessary ellipsis.

This was copied over from SEO so I will prune/refactor after we get the share feature in a good stable place